debug.cabal

```diff
@@ -33,15 +33,20 @@ library
     hs-source-dirs:   src
     build-depends:
         base == 4.*,
+        bytestring,
+        aeson,
         ghc-prim,
         extra,
         containers,
         directory,
         template-haskell,
         open-browser,
+        text,
         uniplate,
+        unordered-containers,
```

Comment:
Given the addition of unordered containers it probably makes sense to stop using containers

Reply:
Agreed